scripts/zones/Yuhtunga_Jungle/mobs/Nasus.lua

```diff
@@ -0,0 +1,14 @@
+-----------------------------------
+-- Area: Yuhtunga Jungle
+--  Mob: Nasus
+-----------------------------------
+local ID = require("scripts/zones/Yuhtunga_Jungle/IDs")
+
+function onMobDisengage(mob)
+    DespawnMob(mob:getID(), 120)
```

Comment:
Could be wrong, but I believe that once this fires, the mob will always despawn at that the end of the timer, even if it re-engages later.

We do have a mobMod for idle despawn, usage looks something like this:
```lua
-- mobs_script.lua
function onMobInitialize(mob)
    mob:setMobMod(tpz.mobMod.IDLE_DESPAWN, 120)
end
```
And it'll handle idle despawn for you automagically after that, without you having to worry about engage/disengage.

Reply:
Nice, I'll use that instead then.
The code I had was something I found on another mob, and just reused it expecting it to be what I needed.